### DISCUSSION TITLE
Blueprint does not allow to add my ccu3

### DISCUSSION THREAD

**Discussion by EddyVanEm** [SukramJ/aiohomematic#685] [Q&A]
Hi,
the blueprint homematicip_local_persistent_notification.yaml does not allow me to add my ccu2 as entity to the blueprint. Can someone please push me in the right direction?

Thanks

**Comment by SukramJ** [comment]
Do you use the latest version of the blueprint?

**Comment by EddyVanEm** [comment]
The one from 24 days ago?

**Reply by SukramJ** [reply]
Yes

**Reply by EddyVanEm** [reply]
That's the one I used.

**Reply by EddyVanEm** [reply]
I just checked. That wasn't the one. I just updated. But I can't import due to errors:
<img width="1379" alt="image" src="https://user-images.githubusercontent.com/34966638/198315940-64c475b2-7b9f-4a19-b5f8-2527cc91bcac.png">

**Reply by SukramJ** [reply]
Then everything should be fine.
There must be an entity `sensor.{your instanz name}` Is it there an enabled.

**Reply by EddyVanEm** [reply]
Sorry, we replied at the same time. Check my previous screenshot.

**Reply by SukramJ** [reply]
This is a validation error. The blueprint is working on my instance.

**Reply by EddyVanEm** [reply]
Ok, seems to work now. Thanks for the help! Much appreciated!

**Reply by SukramJ** [reply]
Any hint what the problem was?

**Reply by EddyVanEm** [reply]
Well first I didn‘t use the most recent version of the blueprint. And the error line in the script I just deleted.

**Reply by SukramJ** [reply]
> And the error line in the script I just deleted.

That shouldn't be an issue. It's just a script validation used as filter.

What are the attributes of your sensor.{your instance name} entity?
